packages/dev/gui/src/2D/controls/control.ts

```diff
@@ -1817,6 +1817,10 @@ export class Control implements IAnimatable {
         this._preMeasure(this._tempPaddingMeasure, context);
 
         this._measure();
+
+        // Let children take some post-measurement actions
+        this._postMeasure(this._tempPaddingMeasure, context);

```

Comment:
Wouldn't it will be better to add observables to track the steps? This way you can add a postMeasure action on existing classes and don't have to override them.  Using this architecture you have to create a new class in order to get the functionality working

Reply:
Yes, 100% agree. however, the intention here is to ONLY complete the initial design. Introduce Observable may also introduce overhead . I can make it easely whil i'm in. let me know.

Reply:
all good here, if it serves the purpose I am fine with it.